### DISCUSSION TITLE
How do I check my translation?

### DISCUSSION THREAD

**Discussion by Leifdin** [lxqt/lxqt#2015] [Q&A] [answered]
Hello, I've recently translated some strings into the Slovak language. I'd like to check my work to see if it works correctly and feels right. Is there a way to check them?

**Comment by stefonarch** [comment, marked as answer]
You can download the *.ts files from github or in weblate for every component, install qt5linguist, open the file with it and hit "release", it will create a file `<component>.qm` which you can copy to` /usr/share/lxqt/translations/<component>/` or `~/.local/share/lxqt/translations/<component>/` which would be better but I didn't test this (edit: tested, doesn't work for pcmanfm but for lxqt components yes).

But as they are many of them this takes time, another way would be compiling from source the component.

Which distro are you using?

**Reply by Leifdin** [reply]
I am using Lubuntu 21.04

**Reply by stefonarch** [reply]
I tested in the meantime, it should work saving the newer *.qm files to `~/.local/share/lxqt/translations/<component>/` so you will not overwrite system files. You'l'l have to create those directories.

**Reply by Leifdin** [reply]
Thank you very much
